docs/snippets/common/test-runner-helper-function.js.mdx

```diff
@@ -1,7 +1,7 @@
 ```js
 // .storybook/test-runner.js
 
-const { getStoryContext } = require('@storybook/test-runner');
+const { getStoryContext, waitForPageReady } = require('@storybook/test-runner');
 
 module.exports = {
```

Comment:
Regarding module.exports, I think ideally our snippets should be in Typescript and all use `export default`, but I think it's also fine to do this at a later stage. I also need to do something similar in the README of the test-runner!